CONTRIBUTING.md

```diff
@@ -12,47 +12,47 @@ For big feature it's advised to raise an issue to discuss it first.
 * To quickly get up and running, you can just run `dotnet build SentryNoMobile.slnf` (you're skipping the mobile targets)
 * To run a full build in Release mode and test, before pushing, run `./build.sh` or `./build.cmd`
 
-## Dependencies
+## Minimal Dependencies
 
 * The latest versions of the following .NET SDKs:
   - [.NET 8.0](https://dotnet.microsoft.com/download/dotnet/8.0)
   - [.NET 6.0](https://dotnet.microsoft.com/download/dotnet/6.0)
 
   *Technically, you only need the full SDK install for the latest version (8.0).  If you like, you can install the smaller ASP.NET Core Runtime packages for .NET 6.0. However, installing the full SDKs will also give you the runtimes.*
 
-  *If using an "Apple silicon" processor (M1 or newer), read [the special instructions below](#special-instructions-for-apple-silicon-cpus).*
+* [`pwsh`](https://github.com/PowerShell/PowerShell#get-powershell) Core version 6 or later on PATH.
 
-* You'll need [`pwsh`](https://github.com/PowerShell/PowerShell#get-powershell) Core version 6 or later on PATH.
```

Comment:
I have bumped into this issue previously and it took me a little while to work out... I think worth leaving this in there.

Reply:
Which one? The `pwsh`? That moved up 2 lines.